src/components/TypeSelect/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function TypeSelect({getTepmeratureType, value}){
+  return(
+    <>
+      <form>
+        <label> Choose a type </label>
+        <select onChange={getTepmeratureType} value={value}>
+          <option value='kelvin'>Kelvin</option>
+          <option value='fahrenheit'>Fahrenheit</option>
+          <option value='celsius'>Celsius</option>
+        </select>
+      </form>
+    </>
+  )
+}
+
+TypeSelect.propTypes = {
+  getTepmeratureType: PropTypes.func.isRequired,
+  value: PropTypes.string
+};
```

Comment:
`value` should be required or have a default value defined in `TypeSelect.defaultValue`